hedera-node/src/main/java/com/hedera/services/txns/schedule/ScheduleSignTransitionLogic.java

```diff
@@ -96,7 +98,11 @@ private void transitionFor(ScheduleSignTransactionBody op) throws Exception {
         }
 
         if (readyForExecution(op.getSchedule())) {
-            // TODO: prepare child tx for execution
+            var transaction = prepareTransaction(store.get(op.getSchedule()));
+            txnCtx.trigger(new TriggeredTxnAccessor(transaction,
+                    store.get(op.getSchedule()).payer().toGrpcAccountId(),
+                    op.getSchedule()));
+            outcome = store.execute(op.getSchedule());
         }
```

Comment:
This method is exactly the same as in the `ScheduleCreateTransitionLogic`. Maybe we could add a new method in `ScheduleReadyForExecution` -> `processExecutionReadiness` (maybe there is a better name for that)